### DISCUSSION TITLE
Add Charity API endpoint and EventSub subscription type

### DISCUSSION THREAD

**Discussion by minimike86** [TwitchIO/TwitchIO#415] [Feature requests]
Hey, 

Are there any plans to add the Charity API endpoint and EventSub subscription type?
https://discuss.dev.twitch.tv/t/new-charity-api-endpoint-and-eventsub-subscription-type-are-now-in-open-beta/40197

channel.charity_campaign.donate
Sends a notification when a user donates to the broadcaster’s charity campaign.

Would be particularly useful

**Comment by chillymosh** [comment]
Yes, these will be added.

**Comment by IAmTomahawkx** [comment]
done in 3b13fe9 and 856d383
